baseframe/forms/validators.py

```diff
@@ -16,16 +16,33 @@
 from ..signals import exception_catchall
 
 
-__all__ = ['OptionalIf', 'OptionalIfNot', 'ValidEmail', 'ValidEmailDomain', 'ValidUrl', 'AllUrlsValid',
-    'ValidName', 'NoObfuscatedEmail', 'ValidCoordinates', 'IsPublicEmailDomain', 'IsNotPublicEmailDomain',
-    # WTForms validators
-    'DataRequired', 'InputRequired', 'Optional', 'Length', 'EqualTo', 'URL', 'NumberRange',
-    'ValidationError', 'StopValidation']
+__local = ['AllUrlsValid', 'IsNotPublicEmailDomain', 'IsPublicEmailDomain', 'NoObfuscatedEmail',
+    'OptionalIf', 'OptionalIfNot', 'RequiredIf', 'ValidCoordinates', 'ValidEmail',
+    'ValidEmailDomain', 'ValidName', 'ValidUrl']
+__imported = [  # WTForms validators
+    'DataRequired', 'EqualTo', 'InputRequired', 'Length', 'NumberRange', 'Optional',
+    'StopValidation', 'URL', 'ValidationError']
+__all__ = __local + __imported
 
 
 EMAIL_RE = re.compile(r'\b[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,63}\b', re.I)
 
 
+class AllowedIf(object):
+    """
+    Validator that makes this field required if some other field is true.
+    """
+    def __init__(self, fieldname, message=None):
+        self.fieldname = fieldname
+        self.message = message or __("This field is not allowed if '{}' is not set".format(self.fieldname))
```

Comment:
We can't display "cfp_start_at" to the user. Extract the label and add it here. Also, you can't have a parameter inside `__()`. The format operation has to be outside.